kimchi/src/prover.rs

```diff
@@ -940,6 +940,11 @@ where
                         .to_chunked_polynomial(index.max_poly_size)
                         .evaluate_chunks(zeta)
                 }),
+                coefficients: array::from_fn(|i| {
```

Comment:
Ah ok so indeed the coefficients are the evaluation on zeta (rows) for the monomial-formed polynomial representing the coefficients (meaning, the `coefficientsm` field of the constraint system). So basically the `coefficients` field inside the ProofEvaluations refers to a "doubly" evaluated version of the coefficients, over the zeta first, and then the outcome over the `pt` above, right?